src/azure-cli-core/azure/cli/core/_profile.py

```diff
@@ -837,10 +843,16 @@ def create_arm_client_factory(credentials):
 
     def find_from_user_account(self, username, password, tenant, resource):
         context = self._create_auth_context(tenant)
-        if password:
-            token_entry = context.acquire_token_with_username_password(resource, username, password, _CLIENT_ID)
-        else:  # when refresh account, we will leverage local cached tokens
-            token_entry = context.acquire_token(resource, username, _CLIENT_ID)
+        try:
+            if password:
+                token_entry = context.acquire_token_with_username_password(resource, username, password, _CLIENT_ID)
+            else:  # when refresh account, we will leverage local cached tokens
+                token_entry = context.acquire_token(resource, username, _CLIENT_ID)
+        except InvalidURL as err:
+            logger.debug('Invalid url when acquiring token with username:%s password:%s\n%s',
+                         username, password, traceback.format_exc())
+            raise ArgumentParseError('Invalid url when acquiring token: %s' %
```

Comment:
Is this a `InvalidArgumentValueError` here？ I am not sure whether the url is provided by users.

Reply:
replaced with `UnknownError`